spec/unit/spark_api/faraday_middleware_spec.rb

```diff
@@ -48,6 +48,13 @@
             "Code": "0000"
           }}'] 
         }
+        stub.get('/badresourcerequest') { [409, {}, '{"D": {
+            "Message": "FlexmlsApiResponse::Errors::Errors", 
```

Comment:
Sorry about this message, we have an open issue somewhere to make that an actual message.